core/src/main/java/com/novoda/noplayer/exoplayer/SecurityDowngradingCodecSelector.java

```diff
@@ -7,22 +7,20 @@
 class SecurityDowngradingCodecSelector implements MediaCodecSelector {
 
     private final InternalMediaCodecUtil internalMediaCodecUtil;
-    private final boolean downgradeSecureDecoder;
 
-    public static SecurityDowngradingCodecSelector newInstance(boolean downgradeSecureDecoder) {
+    public static SecurityDowngradingCodecSelector newInstance() {
         InternalMediaCodecUtil internalMediaCodecUtil = new InternalMediaCodecUtil();
-        return new SecurityDowngradingCodecSelector(internalMediaCodecUtil, downgradeSecureDecoder);
+        return new SecurityDowngradingCodecSelector(internalMediaCodecUtil);
     }
 
-    SecurityDowngradingCodecSelector(InternalMediaCodecUtil internalMediaCodecUtil, boolean downgradeSecureDecoder) {
+    SecurityDowngradingCodecSelector(InternalMediaCodecUtil internalMediaCodecUtil) {
         this.internalMediaCodecUtil = internalMediaCodecUtil;
-        this.downgradeSecureDecoder = downgradeSecureDecoder;
     }
 
     @Override
     public MediaCodecInfo getDecoderInfo(String mimeType, boolean contentRequiresSecureDecoder)
```

Comment:
is `contentRequiresSecureDecoder` not used anymore? if so... kill it? (if swe can, which I don't know)